FirstSolution/Part14Task1/Task.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+
+namespace Part14Task1
+{
+    // 1. Ниже находятся варианты тем. Выберите ту тему, которая вам больше всего нравится.
+    // 1.1 Разработать один из классов на выбор
+    // 1.2 Реализовать не менее пяти закрытых полей (различных типов), представляющих основные
+    // характеристики рассматриваемого класса.
+    // 1.3 Создать не менее трех методов управления классом и методы доступа к его закрытым полям.
+    // 1.4 Создать не менее двух статических полей (различных типов), представляющих
+    // общие характеристики объектов данного класса.
+    // 1.5 Обязательным требованием является реализация нескольких перегруженных конструкторов,
+    // аргументы которых определяются студентом, исходя из специфики реализуемого класса,
+    // а так же реализация конструктора по умолчанию.
+    // 1.6 Создать статический конструктор.
+    // 1.7 Создать массив (не менее 5 элементов) объектов созданного класса.
+
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            Car ford = new Car("Ford", "Blue", 110, 4.12, 1.76);
+            ford.PrintInformation();
+            ford.RepaintCarInRed();
+            ford.AddHorsepower();
+            ford.PrintInformation();
+            Car[] cars =
+            {
+                new Car("Mazda", "Black", 110, 4.5, 1.6),
+                new Car(180, 6.5, 1.9),
+                new Car("Dodge", "Yellow"),
+                new Car("Audi", "Grey", 160, 4.87, 1.78),
+                new Car("BMW", "Black", 150, 4.5, 1.81)
+            };
+        }
+    }
+
+    public class Car
+    {
+        private static int numberOfWheels = 4;
```

Comment:
Try to move initialization to static constructor

Reply:
Done